### ISSUE TITLE
Please upgrade hadolint to >=2.9.2

### ISSUE THREAD

**Issue by DrYak** [megalinter/megalinter#1367]
**Describe the bug**
The version 2.9.1 used by the latest Megalinter is affected by bug [785](https://github.com/hadolint/hadolint/issues/795): false positive on pragmas.

**To Reproduce**
Dockerfile starting with a syntax pragma, for example:
```Dockerfile
# syntax=docker/dockerfile:1.3
ARG test=val
FROM debian:stable
…etc…
```

**Expected behavior**
versions <=2.9.0 and >=2.9.2 correctly recognize the above as valid (it only has comments (with pragma), and `ARG` until the `FROM` section begins).

**Screenshots**
Version 2.9.1 triggers this false positive instead:
> DL3061 error: Invalid instruction order. Dockerfile must begin with FROM, ARG or comment.

**Additional context**
We use mega-linter to automatically check our [bioinformatics pipeline](https://github.com/cbg-ethz/V-pipe) and it is [affected](https://github.com/cbg-ethz/V-pipe/runs/5739622996?check_suite_focus=true) by [this bug](https://github.com/cbg-ethz/V-pipe/blob/master/Dockerfile#L1) as current megalinter uses the affected version 2.9.0 of hadolint.

**Comment by nvuillam** [comment]
It's on the way :)
